### PR TITLE
create index.html

### PR DESCRIPTION
**Pull request by gaditi123**
Added the header portion

### CODE REVIEW

**Comment by killbotXD** [comment]
@gaditi123 always remember. commit message should contain the changes that you have made. You should not name all the commit messages as `update index.html`

**Comment by gaditi123** [comment]
> @gaditi123 always remember. commit message should contain the changes that you have made. You should not name all the commit messages as `update index.html`

i've written the changes in the further message box

File: index.html [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+  
+<button class="btn btn-primary dropdown-toggle mr-4" type="button" data-toggle="dropdown"
+  aria-haspopup="true" aria-expanded="false">Year</button>
+
+<div class="dropdown-menu">
+  
+</div>
+
+</div>
+  <div class ="Album-holder">
```
Comment (killbotXD) [inline index.html]: probably redundant

File: index.html [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+  
+  <div class = "cards-holder">
+  
+  </div>
+  
+ </div>
+ 
+  </div>
+  
+  </div>
```
Comment (killbotXD) [inline index.html]: redundant

**Pull request merged**